src/generator/app.js

```diff
@@ -21,12 +21,20 @@ app.set('port', (process.env.PORT || 5000));
 
 // Use the www folder as static frontend
 app.use('/', express.static(__dirname + '/www'));
+app.use('/live/preview', express.static(__dirname + '/../../dist'));
 
-app.post('/generate', uploadedFiles, function(req, res, next) {
+app.post('/live', uploadedFiles, function(req, res) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Using "FunctionExpression" is not allowed.](https://www.codacy.com/app/triplez/open-event-webapp/file/2758438178/issues/source?bid=3233857&fileBranchId=3359620#l26)